### ISSUE TITLE
C-hw-i-l as a bundle

### ISSUE THREAD

**Issue by willbasky** [vacuumlabs/cardano-hw-interop-lib#21]
Are anywhere there a single-file.js with bundled `cardano-hw-interop-lib` that I can add to frontend client?

**Comment by janmazak** [comment]
We are not sure what you are trying to achieve. Perhaps you can use webpack?
This library only checks if a transaction conforms to CIP-21 (and can be used for transaction parsing, e.g. in cardano-hw-cli), it does not do any actual work (like transaction signing), nor can it initiate any transaction signing.

**Comment by willbasky** [comment]
I need to adapt a transaction in the way that it was consumed by hard wallets. On client side. My client is written with Reflex library, I could use Haskell library for that purposes but I didn't find any. Therefore I have to bundle your library and load as single `js` file. 
As I saw this library can transform tx to cip-21 format, not only validate tx.

So, that's why I open issue. Maybe you have the bundle somewhere or at least webpack config.

**Comment by janmazak** [comment]
Yes, it can transform a transaction too. Right now, we are busy with adding Conway features, which is a very high priority, but perhaps you can prepare a PR with the webpack config/setup and we can add it to the library?

**Comment by willbasky** [comment]
Sure, when I have any good solution for that I commit it.
